pwa-devdocs/scripts/lint/index.js

```diff
@@ -7,11 +7,26 @@ const config = {
     basePath: path.join(__dirname, '../../src')
 };
 
-// If a single file is specified, only run tests on that file
+// If a file or directory is specified, only run tests on that file or directory
 const filepath = process.argv[2];
 if (filepath) {
     const fullPath = path.join(__dirname, '..', '..', filepath);
-    lintFile(fullPath);
+
+    fs.stat(fullPath, (error, stats) => {
+        if (error) {
```

Comment:
Even better! I think this removes the need for the default `else` case 👍

Reply:
Oh oops, I merged this before letting you address this comment 😝 .

Reply:
that's alright. it still works and the default else will just be used to catch invalid types